pyuvdata/tests/test_miriad.py

```diff
@@ -151,6 +167,57 @@ def test_poltoind():
     nt.assert_raises(ValueError, miriad._pol_to_ind, pol_arr[0])
 
 
+def test_miriad_extra_keywords():
+    uv_in = UVData()
+    uv_out = UVData()
+    miriad_file = os.path.join(DATA_PATH, 'zen.2456865.60537.xy.uvcRREAA')
+    testfile = os.path.join(DATA_PATH, 'test/outtest_miriad.uv')
+
+    # check for warnings & errors with extra_keywords that are dicts, lists or arrays
+    uvtest.checkWarnings(uv_in.read_miriad, [miriad_file],
+                         known_warning='miriad')
+    uv_in.extra_keywords['testdict'] = {'testkey': 23}
+    uvtest.checkWarnings(uv_in.check, message=['testdict in extra_keywords is a '
+                                               'list, array or dict'])
+    nt.assert_raises(TypeError, uv_in.write_miriad, testfile, clobber=True,
+                     run_check=False)
+
+    uvtest.checkWarnings(uv_in.read_miriad, [miriad_file],
```

Comment:
no, the things that gets added previously just needs to be removed. I'll fix that.